tests/unit_tests/test_game_rater.py

```diff
@@ -1,90 +1,79 @@
 import pytest
-from trueskill import Rating
 
 from server.games.game_results import GameOutcome
-from server.rating import RatingType
+from server.games.typedefs import TeamRatingSummary
+from server.rating import Rating, RatingType
 from server.rating_service.game_rater import GameRater
-from server.rating_service.typedefs import TeamRatingData
-
-
-class MockPlayer:
-    ratings = {RatingType.GLOBAL: (1500, 500), RatingType.LADDER_1V1: (1500, 500)}
+from server.rating_service.typedefs import GameRatingSummary
 
 
 @pytest.fixture
 def rating_data_1v1():
     player1_id = 1
-    player1_rating = Rating(1500, 500)
-    player1_outcome = GameOutcome.VICTORY
-
     player2_id = 2
-    player2_rating = Rating(1400, 400)
-    player2_outcome = GameOutcome.DEFEAT
 
-    return [
-        TeamRatingData(player1_outcome, {player1_id: player1_rating}),
-        TeamRatingData(player2_outcome, {player2_id: player2_rating}),
-    ]
+    ratings = {
+        player1_id: Rating(1500, 500),
+        player2_id: Rating(1400, 400),
+    }
+
+    return GameRatingSummary(
+        1,
+        RatingType.GLOBAL,
+        [
+            TeamRatingSummary(GameOutcome.VICTORY, {player1_id}, []),
+            TeamRatingSummary(GameOutcome.DEFEAT, {player2_id}, [])
+        ]
+    ), ratings
 
 
 @pytest.fixture
 def rating_data_2v2():
     player1_id = 1
-    player1_rating = Rating(1500, 500)
-
     player2_id = 2
-    player2_rating = Rating(1400, 400)
-
     player3_id = 3
-    player3_rating = Rating(1300, 300)
-
     player4_id = 4
-    player4_rating = Rating(1200, 200)
 
-    team1_outcome = GameOutcome.VICTORY
-    team2_outcome = GameOutcome.DEFEAT
+    ratings = {
+        player1_id: Rating(1500, 500),
+        player2_id: Rating(1400, 400),
+        player3_id: Rating(1300, 300),
+        player4_id: Rating(1200, 200)
+    }
 
-    return [
-        TeamRatingData(
-            team1_outcome, {player1_id: player1_rating, player2_id: player2_rating}
-        ),
-        TeamRatingData(
-            team2_outcome, {player3_id: player3_rating, player4_id: player4_rating}
-        ),
-    ]
+    return GameRatingSummary(
+        1,
+        RatingType.GLOBAL,
+        [
+            TeamRatingSummary(GameOutcome.VICTORY, set((player1_id, player2_id)), []),
+            TeamRatingSummary(GameOutcome.DEFEAT, set((player3_id, player4_id)), [])
```

Comment:
Maybe a question of preference, but you could use {} here too. Then it is also more consistent with the fixture above

Reply:
Yea I like that better, I just wasn't thinking of it at the time.